src/interface/stemplot.jl

```diff
@@ -0,0 +1,133 @@
+"""
+`stemplot(v; nargs...)`->` Plot`
+
+Description
+===========
+
+Draws a stem leaf plot of the given vector 'v'.
+
+Usage
+=====
+
+`stemplot(v)`
+
+function stemplot(
+                  v::Vector,
+                  symbol::AbstractString, 
+                  scale::Int64)
+Arguments
+=========
+
+-**`v`** : Vector for which the stem leaf plot should be computed
+-**`symbol`: Symbol for break between stem and leaf. default = "|"
+-**`expand`**: Bool for expanded (true) plot or collapsed (false). default = true
+-**`color`**: Set color of plot. default = white TODO
+
+Results
+=======
+
+A plot of object type 
+
+TODO
+
+Author(s)
+========
+- Iain Dunning (Github: https://github.com/IainNZ)
+- Christof Stocker (Github: https://github.com/Evizero)
+- Kenta Sato (Github: https://github.com/bicycle1885)
+- Alex Hallam (Github: https://github.com/alexhallam)
+
+Examples
+========
+
+`stemplot(rand(1:100,80))`
+
+`stemplot(rand(-100:100,300))`
+
+`stemplot(randn(50),scale = 1)`
+
+`stemplot(randn(50),scale = .1)`
+
+`stemplot(rand(1:59, 200),divider = ":")`
```

Comment:
I'd prefer a single example, but with the output added. (example: https://github.com/Evizero/UnicodePlots.jl/blob/master/src/interface/barplot.jl#L59-L72)